mqtt/docker/linux/amd64/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM busybox
+# Use the same base image as prod edgehub images
+ARG base_tag=3.1.4-alpine3.11
+FROM mcr.microsoft.com/dotnet/core/aspnet:${base_tag}
```

Comment:
If we also use hub base here then the test image approach is the same across platforms and the environments will always be the same. The tradeoff is an increased image size. I don't have a strong opinion here so it is your choice.